app/views/referee_mailer/reference_request_email.text.erb

```diff
@@ -1,20 +1,10 @@
 Dear <%= @reference.name %>,
 
-# Give a reference for <%= @candidate_name %>
+<%= @candidate_name %> put us in touch with you to get a reference for their teacher training application.
 
-<%= @candidate_name %> put us in touch with you to get a reference for their teacher training application. They applied to:
+Please use the link below to give a reference as soon as possible
 
-<% @application_form.application_choices.each do |application_choice| %>
-* <%= application_choice.provider.name %> - <%= application_choice.course.name %>
-<% end %>
-
-Please use the link below to give a complete reference as soon as possible.
-
-<%= @google_form_url %>
-
-If we don’t hear from you within 5 working days we’ll send you a reminder.
-
-If you won’t give <%= @candidate_name %> a reference, please let us know as soon as possible by emailing: [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk)
+<%= @reference_link %>
```

Comment:
Do we not need to add the paragraph describing what to do when they don't want the reference here?

Reply:
Yes, I resolved this in 4354366

Reply:
Acctually we added that iten to the https://trello.com/c/B4tokVQU/§, because we were aware this PR was getting long.